Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -1174,3 +1174,5 @@ newBatch=New batch
 newUser=New user
 newUserGroup=New user group
 newLdapGroup=New LDAP group
+locked=Locked
```

Comment:
It would be useful to keep alphabetical order of properties in messages files. It would help to avoid double messages for the same purpose.